ffi/Cargo.toml

```diff
@@ -15,12 +15,13 @@ crate-type = ["cdylib", "staticlib"]
 [dependencies]
 tracing = "0.1"
 url = "2"
-deltakernel = { path = "../kernel", features = ["developer-visibility"] }
+delta_kernel = { path = "../kernel", features = ["developer-visibility"] }
 
 [build-dependencies]
 cbindgen = "0.26.0"
+delta_kernel = { path = "../kernel" }
```

Comment:
why build dep?

Reply:
oops, not sure how that snuck in. thanks.